src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
So that means the old DELight output variable name is still getting setup.  That needs to go, unlessssss, you want to   keep Daylighting Reference Point Illuminance and set it up for all reference points.  So, the same value could be reported as <RefpointName>:Daylighting Reference Point Illuminance and <Daylighting_ctrl name>:Daylighting Reference Point 1 Illuminance.  Actually, that's probably better.